docs/UserGuide.md

```diff
@@ -348,10 +348,41 @@ DonnaFin data are saved in the hard disk automatically after any command that ch
 
 ### Editing the data file
 
-DonnaFin data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+DonnaFin's data is saved as a JSON file `[JAR file location]/data/donnafin.json`. Advanced users are welcome to update data directly by editing that data file.
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
 If your changes to the data file makes its format invalid, DonnaFin will discard all data and start with an empty data file at the next run.
+
+Original `donnaFin.json` format           |  Invalid `donnaFin.json` format due to direct editing
+:-------------------------:|:-------------------------:
+![DataSetAlexYeoh](images/DataSetAlexYeoh.png) | ![DataSetAlexYeohWrongFormat](./images/DataSetAlexYeohWrongFormat.png)
+
+As seen in the above side by side comparsion, the first figure show the original .json data 
+regarding the client Alex Yeoh. All attributes are in the correct format. However, if you do tamper with the 
+json file directly and change one of the attributes to an invalid format, in this case the 
+total value of assets(supposed to be prefixed with a $ to indicate that it is a monetary value), DonnaFin will
+discard all the data and start with an empty data file as seen below.
+When this happens however, to prevent total loss of your data, we do not delete it right away.
+
+DonnaFinGui           |  `donnaFin.json`
+:-------------------------:|:-------------------------:
+![NoUser](images/EmptyGUIForDonnaFin.png) | ![NoUserJson](./images/DataSetFullAlexYeoh.png)
+
+The follow error message is displayed as well:
+![InvalidFormat](images/InvalidFormatError.png)
+
+When this happens however, to prevent total loss of your data, we do not delete it right away.
+
+
+Only when a new add command is run, will DonnaFin assume this is the intended action and proceed to cleanly wipe
+the DonnaFin.json and populate with the new client as seen below.
+
+
+DonnaFinGui           |  `DonnaFin.json`
+:-------------------------:|:-------------------------:
+![DonnaFinSingleUser](images/DonnaFinSingleUser.png) | ![DonnaFinJsonSingleUser](./images/DonnaFinJsonSingleUser.png)
+
+
```

Comment:
I do not believe this part is necessary as **_any_** command run would cause DonnaFin to wipe out the data. As such, stating that any valid input from the user would cause all data to be deleted should suffice.

Reply:
I just change line 377 then to any input?

Reply:
I tried it out, the input has to be a valid command. so change 377 to any valid command is run. and you can delete everything below i don't think it is necessary.